src/components/common/styles.js

```diff
@@ -61,3 +61,161 @@ export const StyledDot = styled.div`
   height: 0.5rem;
   border-radius: 1rem;
 `;
+
+/* login page styled components */
+export const StyledWrapper = styled.section`
+  background-color: #FFFFFF;
+  width: 30rem;
+  margin: 7rem 0 0 10rem;
+  position: fixed;
+  border-radius: 0.75rem;
+  box-shadow: 0px 11px 35px 2px rgba(0, 0, 0, 0.14);
+  align-items: center;
+  text-align: center;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Is this necessary? I thought the global font was already set.

Reply:
You're right, not necessary.